models/config.py

```diff
@@ -35,12 +35,12 @@ class VLMConfig:
     mp_pixel_shuffle_factor: int = 2
 
     vlm_load_backbone_weights: bool = True
-    vlm_checkpoint_path: str = 'nanoVLM.pth'
+    vlm_checkpoint_path: str = 'checkpoints/nanoVLM-222M'
 
 
 @dataclass
 class TrainConfig:
-    lr_mp: float = 1e-4
+    lr_mp: float = 1e-3
```

Comment:
is this change intentional? (seems unrelated to the PR)

```suggestion
    lr_mp: float = 1e-4
```

Reply:
Is an artifact of me testing the training, should not be a big issue